### PR TITLE
Add Malcolm Tucker insults, plus some text fixes

### PR DESCRIPTION
**Pull request by ivystopia**
Added some quotes at the end.

First commit is just fixing some grammar, dupes and wording.

### CODE REVIEW

File: data/attacks/insult.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
             "What are you doing here? Did someone leave your cage open?",
             "It's better to let someone think you are an Idiot than to open your mouth and prove it.",
             "Shock me, say something intelligent.",
-            "Why don't you slip into something more comfortable -- like a coma.",
```
Comment (ivystopia) [inline data/attacks/insult.json:54]: this was a duplicate

File: data/attacks/insult.json [hunk 2]
```diff
@@ -66,7 +65,7 @@
             "At least when I do a handstand my stomach doesn't hit me in the face.",
             "I'll never forget the first time we met, although I'll keep trying.",
             "You're so dumb, your dog teaches you tricks.",
-            "Don't feel sad, don't feel blue, Frankenstein was ugly too.",
+            "Don't feel sad, don't feel blue, Frankenstein's monster was ugly too.",
```
Comment (ivystopia) [inline data/attacks/insult.json:68]: More accurate now, AND the meter sounds better.

File: data/attacks/insult.json [hunk 3]
```diff
@@ -97,14 +96,14 @@
...
-            "Everyone who ever loved you was wrong.",
+            "Everyone who ever loved you was lying.",
             "You're so ugly Hello Kitty said goodbye to you.",
             "You only annoy me when you're breathing.",
             "We all sprang from apes, but you didn't spring far enough.",
             "I've seen people like you, but I had to pay admission!",
             "If what you don't know can't hurt you, you're invulnerable.",
             "So you've changed your mind, does this one work any better?",
-            "I heard you took an IQ test and they said you're results were negative.",
+            "I heard you took an IQ test and they said your results were negative.",
```
Comment (ivystopia) [inline data/attacks/insult.json:106]: ironic

**Comment by linuxdaemon** [comment]
When modifying a JSON data file, please run the `format_json.py` script in the root of the repository and commit the resulting version of your data file. This ensure that all of the data files are formatted in a similar fashion.

**Comment by ivystopia** [comment]
@linuxdaemon I have now done so, the trailing newline was removed. I've ammended the commit and force pushed.

**Pull request merged**